From 8b600ad3994fb3a66ece34afdf072fb28e46c981 Mon Sep 17 00:00:00 2001
From: placiano <placiano80@gmail.com>
Date: Tue, 1 Sep 2015 16:26:48 +0200
Subject: [PATCH] SM-N910F_511: init.carrier.rc patch

---
 SM-N910F_511/init.carrier.rc | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/SM-N910F_511/init.carrier.rc b/SM-N910F_511/init.carrier.rc
index 3f88b7a..e6f5f1b 100755
--- a/SM-N910F_511/init.carrier.rc
+++ b/SM-N910F_511/init.carrier.rc
@@ -164,12 +164,12 @@ service tbased /system/bin/tbased
     oneshot
 
 # SSWAP
-service swapon /sbin/sswap -s
-    class core
-    user root
-    group root
-    seclabel u:r:sswap:s0
-    oneshot
+#service swapon /sbin/sswap -s
+#    class core
+#    user root
+#    group root
+#    seclabel u:r:sswap:s0
+#    oneshot
 
 # Bluetooth dhcpcd config
 service dhcpcd_bt-pan /system/bin/dhcpcd -ABKL
--
libgit2 0.22.2

