From 06f6d53c553333e110c2904b644d47c9a8c5599c Mon Sep 17 00:00:00 2001
From: placiano <placiano80@gmail.com>
Date: Tue, 1 Sep 2015 16:15:57 +0200
Subject: [PATCH] SM-N910F_511: ueventd.rc patch

---
 SM-N910F_511/ueventd.rc | 2 ++
 1 file changed, 2 insertions(+), 0 deletions(-)

diff --git a/SM-N910F_511/ueventd.rc b/SM-N910F_511/ueventd.rc
index 6c98975..a0d4a8f 100644
--- a/SM-N910F_511/ueventd.rc
+++ b/SM-N910F_511/ueventd.rc
@@ -8,6 +8,8 @@ subsystem adf
 /dev/tty                  0666   root       root
 /dev/random               0666   root       root
 /dev/urandom              0666   root       root
+/dev/frandom              0444   root       root
+/dev/erandom              0444   root       root
 # Make HW RNG readable by group system to let EntropyMixer read it.
 /dev/hw_random            0440   root       system
 /dev/ashmem               0666   root       root
--
libgit2 0.22.2

