From 8e645fd718c495ba8ddd1e7daf7b9618f974a9ef Mon Sep 17 00:00:00 2001
From: placiano <placiano80@gmail.com>
Date: Wed, 2 Sep 2015 02:08:10 +0200
Subject: [PATCH] SM-N910T_511: default.prop patch

---
 SM-N910T_511/default.prop | 15 ++++++++++-----
 1 file changed, 10 insertions(+), 5 deletions(-)

diff --git a/SM-N910T_511/default.prop b/SM-N910T_511/default.prop
index d8bbf70..a91316a 100644
--- a/SM-N910T_511/default.prop
+++ b/SM-N910T_511/default.prop
@@ -10,15 +10,20 @@ persist.cne.feature=0
 persist.radio.lte_vrte_ltd=1
 persist.radio.add_power_save=1
 persist.data.netmgrd.qos.enable=false
-persist.security.ams.enforcing=1
-ro.secure=1
+persist.security.ams.enforcing=0
+ro.secure=0
 ro.allow.mock.location=0
-ro.debuggable=0
-ro.adb.secure=1
+ro.debuggable=1
+ro.adb.secure=0
 ro.zygote=zygote32
-persist.sys.usb.config=mtp
+persist.sys.usb.config=mtp,adb
 dalvik.vm.dex2oat-Xms=64m
 dalvik.vm.dex2oat-Xmx=512m
 dalvik.vm.image-dex2oat-Xms=64m
 dalvik.vm.image-dex2oat-Xmx=64m
 ro.dalvik.vm.native.bridge=0
+ro.securestorage.support=false
+wlan.wfd.hdcp=disable
+androidboot.selinux=0
+ro.config.knox=0
+ro.config.tima=0
--
libgit2 0.22.2

