From 727e8a041602272675a149211071d80d2cfb0b6f Mon Sep 17 00:00:00 2001
From: placiano <placiano80@gmail.com>
Date: Thu, 18 Jun 2015 15:49:22 +0200
Subject: [PATCH] default.prop.patch

---
 SM-N910F/default.prop | 33 ++++++++++++++++++++++++++++-----
 1 file changed, 28 insertions(+), 5 deletions(-)

diff --git a/SM-N910F/default.prop b/SM-N910F/default.prop
index defbff1..eb64764 100644
--- a/SM-N910F/default.prop
+++ b/SM-N910F/default.prop
@@ -2,15 +2,38 @@
 # ADDITIONAL_DEFAULT_PROPERTIES
 #
 persist.cne.feature=0
-persist.security.ams.enforcing=1
-ro.secure=1
+persist.security.ams.enforcing=0
+ro.secure=0
 ro.allow.mock.location=0
-ro.debuggable=0
-ro.adb.secure=1
+ro.debuggable=1
+ro.adb.secure=0
 ro.zygote=zygote32
-persist.sys.usb.config=mtp
+persist.sys.usb.config=mtp,adb
+ro.securestorage.support=false
 dalvik.vm.dex2oat-Xms=64m
 dalvik.vm.dex2oat-Xmx=512m
 dalvik.vm.image-dex2oat-Xms=64m
 dalvik.vm.image-dex2oat-Xmx=64m
+dalvik.vm.image-dex2oat-filter=everything
+dalvik.vm.dex2oat-filter=everything
 ro.dalvik.vm.native.bridge=0
+wlan.wfd.hdcp=disable
+androidboot.selinux=0
+ro.config.knox=0
+ro.config.tima=0
+ro.config.dha_cached_max=15
+ro.config.dha_empty_max=42
+ro.config.dha_empty_init=42
+ro.config.dha_th_rate=2.3
+ro.config.dha_lmk_scale=0.545
+ro.config.sdha_apps_bg_max=70
+ro.config.oomminfree_high=7628,9768,11909,14515,16655,20469
+windowsmgr.max_events_per_sec=90
+ro.min_pointer_dur=8
+ro.max.fling_velocity=12000
+ro.min.fling_velocity=8000
+video.accelerate.hw=1
+debug.performance.tuning=1
+persist.service.lgospd.enable=0
+persist.service.pcsync.enable=0
+touch.pressure.scale=0.003
\ No newline at end of file
--
libgit2 0.22.2

