From 7a8ece57cea252acbc49133ce67559f21650702d Mon Sep 17 00:00:00 2001
From: placiano <placiano80@gmail.com>
Date: Sat, 23 May 2015 16:55:45 +0200
Subject: [PATCH] patch file_contexts

---
 file_contexts | 2 ++
 1 file changed, 2 insertions(+), 0 deletions(-)

diff --git a/file_contexts b/file_contexts
index 07b67bf..fb90683 100644
--- a/file_contexts
+++ b/file_contexts
@@ -112,6 +112,8 @@
 /dev/uinput		u:object_r:uhid_device:s0
 /dev/uio[0-9]*		u:object_r:uio_device:s0
 /dev/urandom		u:object_r:urandom_device:s0
+/dev/erandom		u:object_r:urandom_device:s0
+/dev/frandom		u:object_r:random_device:s0
 /dev/usb_accessory	u:object_r:usbaccessory_device:s0
 /dev/vcs[0-9a-z]*	u:object_r:vcs_device:s0
 /dev/video[0-9]*	u:object_r:video_device:s0
--
libgit2 0.22.2

