From 6cf208dd2ae89bcaf27a60b7ab95aa4cf0937fec Mon Sep 17 00:00:00 2001
From: placiano <placiano80@gmail.com>
Date: Sun, 24 May 2015 03:39:49 +0200
Subject: [PATCH] patch init.target.rc

---
 init.target.rc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/init.target.rc b/init.target.rc
index 2acea6c..53890ab 100755
--- a/init.target.rc
+++ b/init.target.rc
@@ -225,8 +225,8 @@ on property:init.svc.bootanim=stopped
 on property:sys.qseecomd.enable=true
     start imsqmidaemon
     insmod /system/lib/modules/adsprpc.ko
-    insmod /system/lib/modules/mhi.ko
-    write /dev/kmsg "insmod mhi.ko"
+   # insmod /system/lib/modules/mhi.ko
+   # write /dev/kmsg "insmod mhi.ko"
 
 service imsqmidaemon /system/bin/imsqmidaemon
     class main
--
libgit2 0.22.2

